action.yml

```diff
@@ -41,6 +41,9 @@ inputs:
   skip_if_plasmic:
     description: "Skip build if last commit was authored by Plasmic"
     required: false
+  localization_strings:
```

Comment:
I think it should be `localized_strings`

Reply:
I realize that I started calling it 'localized'...  but the sync command has it as 'localization-strings':
https://docs.plasmic.app/learn/localization-frameworks/

I've updated all cases to be localization_strings_format.  Happy to go with 'localized' if you prefer.
https://github.com/plasmicapp/plasmic-action/pull/3/commits/7e61af0df6cd55eeed2370b5bb7031ec393b4d79#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R44